integration/tests/basic/rpc/test_rpc_get_logs.py

```diff
@@ -287,65 +293,6 @@ def test_eth_get_logs_list_of_addresses(self, event_caller_contract):
                                    "logIndex", "transactionLogIndex"])
             assert_fields_are_boolean(event, ["removed"])
 
-    @pytest.mark.parametrize(
```

Comment:
what difference in eth_getLogs and neon_getLogs? Because right now you don't check specific cases for neon_getLogs (if they exist)

Reply:
The difference in response fields:
```
neon
{'jsonrpc': '2.0', 'id': 22, 'result': [{'removed': False, 'topics': ['0x6da5728c22b39e50f4127d37599a426309bb6695416121047f26c997526b4f9a', '0x50c9e2ea4688fab9f064eb404c4b148933077aa010cffb4d73b1060a9b7d1458'], 'data': '0x', 'blockNumber': '0xf4bc67a', 'transactionHash': '0xcadb505d0f98081bbaf3313b0f350490bedeca9af2164c6a93a2d6c4f9bd05a0', 'transactionIndex': '0x0', 'transactionLogIndex': '0x0', 'logIndex': '0x0', 'address': '0xc6d9b463a3ed7a2413d8b547dbe8421d38e4dcae', 'neonEventOrder': 2, 'neonEventLevel': 1, 'neonSolHash': '2n8kdr1W4vq9ChdoGbwngeb3hj8s7VFck3BjpadBTFwCjwhWC9MnCka5vhdrKC3mCo5HNoU47P6rRsR8xLKSntjz', 'neonIxIdx': 2, 'neonInnerIxIdx': None, 'blockHash': '0x3d67b306fa58bb5a84a6f0819beff939d69724d1252f6e7c0fea8b235f2eabed'}, {'removed': False, 'topics': ['0x6da5728c22b39e50f4127d37599a426309bb6695416121047f26c997526b4f9a', '0x50c9e2ea4688fab9f064eb404c4b148933077aa010cffb4d73b1060a9b7d1458'], 'data': '0x', 'blockNumber': '0xf4bc67e', 'transactionHash': '0xe228cc9e370e84b2c5066420850d5b8b398f8cb030687de53de45cdef1eef6a2', 'transactionIndex': '0x0', 'transactionLogIndex': '0x0', 'logIndex': '0x0', 'address': '0x3fd5733d4d38de1c55e15fad8f34765bead26d51', 'neonEventOrder': 2, 'neonEventLevel': 1, 'neonSolHash': '3isk1m6mpCvCrVFgMfUYsFamtX4kvf7nCazerGkf3rxC4BwrWXNFzRGagPz8SxhUsdMjoX2izrux2P48KacrBou4', 'neonIxIdx': 2, 'neonInnerIxIdx': None, 'blockHash': '0xa332d8539f81c5ecb85d04278695eff5660d29c20c2f7eb92c4d8167b132b016'}]}

eth
{'jsonrpc': '2.0', 'id': 38, 'result': [{'removed': False, 'topics': ['0x6da5728c22b39e50f4127d37599a426309bb6695416121047f26c997526b4f9a', '0x1f35defa9887d33b3a9648944844753c0cb9c1a108925817c890e1af5ad9e9e7'], 'data': '0x', 'blockNumber': '0xf4bc773', 'transactionHash': '0x81c1997bae5be5b62a244ef5bfdaef9f6334009ead58858b00f36996754ac233', 'transactionIndex': '0x0', 'transactionLogIndex': '0x0', 'logIndex': '0x0', 'address': '0x307f38afb59e14075f6d25c7c6dce0b2e539c3d8', 'blockHash': '0x05b61f5374c411195bf01e27688d4f4d87c61f36cbd740a9b5777ebfe40050e5'}, {'removed': False, 'topics': ['0x6da5728c22b39e50f4127d37599a426309bb6695416121047f26c997526b4f9a', '0x1f35defa9887d33b3a9648944844753c0cb9c1a108925817c890e1af5ad9e9e7'], 'data': '0x', 'blockNumber': '0xf4bc778', 'transactionHash': '0x4353fc0dcbe17c1e7921b55d8448f89ca5f964aa093cce3a5d67b063f078b279', 'transactionIndex': '0x0', 'transactionLogIndex': '0x0', 'logIndex': '0x0', 'address': '0x893c9de522793a8601b68be3292a2bc7515151bb', 'blockHash': '0xe221e997d0676078d79f2e594b76393c4f4e8d9699525bcba79aa6a3de94409e'}]}
```

I added additional checks for them.